cmake/plugin_loader-config.cmake.in

```diff
@@ -0,0 +1,14 @@
+@PACKAGE_INIT@
```

Comment:
This file isn't needed, it can be created automatically by RICB now: https://github.com/ros-industrial/ros_industrial_cmake_boilerplate#configure-pure-cmake-package

Reply:
Btw, the version check should actually be 3.9 and all current Ubuntu distros use a higher version.